# IGV
(1) load fai, fna, gff, and big wig files in that order
(2) you can adjust the height of your tracks (80 is a good parameter)
(3) color code by sample
(4) can get subtract your sample from the control to get differences in the peaks
(5) you can also note the samples that did not work 
(6) when you have mapped your reads and are inputing the big wig files, all you need to load is the gff file that corresponds with your genome
